gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/map/AsString.feature

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+@StepClassMap @StepAsString
```

Comment:
i think it would be good to include tests for all the graph element types to ensure consistent `toString()` stuff is happening.

Reply:
I added some tests in https://github.com/apache/tinkerpop/pull/2205/commits/8eb1cf99d3393c35ad73c5516e6ac15697dcf973, and I had to add new regex parsing to get string versions of vertex/edge/vertex properties, is that the correct way to approach asserting string results for these elements?